bin/shared-setup

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
nit: Do you mind calling this script `setup-shared`? It will help keep them together when sorting...

Reply:
This makes sense for the app but you need them for running some of the scripts in `bin`